src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -686,6 +686,103 @@ bool AdaptDispatch::EraseInLine(const DispatchTypes::EraseType eraseType)
     }
 }
 
+// Routine Description:
+// - Selectively erases unprotected cells in an area of the buffer.
+// Arguments:
+// - textBuffer - Target buffer to be erased.
+// - eraseRect - Area of the buffer that will be affected.
+// Return Value:
+// - <none>
+void AdaptDispatch::_SelectiveEraseRect(TextBuffer& textBuffer, const til::rect& eraseRect)
+{
+    if (eraseRect.left < eraseRect.right && eraseRect.top < eraseRect.bottom)
```

Comment:
You could just write `if (eraseRect)` here. The `operator bool` does exactly this, plus it ensures that all coordinates are >= 0, which seems to me like we might want in this code.

Reply:
Oh that's neat. I'll definitely do that.